.remarkrc.mjs

```diff
@@ -76,6 +76,7 @@ const retextPreset = [
 				'type',
 				'validate',
 				'however',
+				'relative-to',
```

Comment:
```suggestion
				'relative to',
```

Reply:
it accepts rule IDs which can't contain spaces AFAIK